migrations/2021-02-27-job-add-is-application-page-active-field.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Add isApplicationPageActive field to the Job model.
+ */
+
+module.exports = {
+  up: queryInterface => {
+    return Promise.all([
+      queryInterface.sequelize.query('ALTER TABLE bookings.jobs ADD is_application_page_active BOOLEAN NOT NULL DEFAULT false'),
+      queryInterface.sequelize.query('UPDATE bookings.jobs SET is_application_page_active=false WHERE is_application_page_active is NULL'),
```

Comment:
I think we don't need this second query `UPDATE bookings.jobs SET is_application_page_active=false WHERE is_application_page_active is NULL` as the previous one would already set all the values bye default, and it also sets `NOT NUL` so there cannot be null values.

But it doesn't harm either.